generator/annual-report/sig_report.tmpl

```diff
@@ -16,13 +16,34 @@
 
 {{$releases := getReleases}}
 {{$owingsig := .Dir}}
+{{$stagedKeps := filterKEPs $owingsig $releases}}
+{{$alphaKeps := index $stagedKeps "alpha"}}
+{{$betaKeps := index $stagedKeps "beta"}}
+{{$stableKeps := index $stagedKeps "stable"}}
+<!--
+   Note: This list is generated from the KEP metadata in k/enhancements repository.
+      If you want find any discrepancy in the generated list here, please check the KEP metadata.
+      Please raise an issue in k/community if the KEP metadata is correct.
```

Comment:
Is the suggestion is to raise an issue in k/community "when the KEP metadata is matching the k/enhancements repo, but there is still a data discrepancy in the generated report"?

 If so, the current wording read a bit unclear.

Reply:
Yes. Thanks @palnabarun